tests/publiccloud/xfsprepare.pm

```diff
@@ -47,7 +47,7 @@ sub ensure_user_exists {
 sub install_xfstests {
     my ($repo) = @_;
     zypper_ar($repo, name => "filesystems");    # Add filesystem repository, which contains the xfstests
-    add_suseconnect_product(get_addon_fullname('phub')) if is_sle;    # packagehub is required for dbench (required for e.g. generic/241)
+    add_suseconnect_product(get_addon_fullname('phub'), undef, undef, undef, 300, 1) if is_sle; # packagehub is required for dbench (required for e.g. generic/241)
```

Comment:
for now we have confirmed problems only for Azure . let's keep it this way . please add logic which will increase timeout only for Azure

Reply:
This is not changing the behavior or the flow, it only increases the timeout. I don't think it harms to keep it for all cases.

Reply:
currently we have ticket for handling timeout for Azure only . we not working anyhow on handling timeouts for Amazon and GCP . The harm which I see is that when problem popup in GCP or Amazon we won't be able to detect this because timeout was already increased for Amazon and GCP .